src/helper/math.js

```diff
@@ -27,9 +27,79 @@ const snapDeltaToAngle = function (delta, snapAngle) {
     return new paper.Point(dirx * d, diry * d);
 };
 
+const _getDepth = function (item) {
+    let temp = item;
+    let depth = 0;
+    while (!(temp instanceof paper.Layer)) {
+        depth++;
+        if (temp.parent === null) {
+            // This item isn't attached to a layer, so it's not on the canvas and can't be compared.
+            return null;
+        }
+        temp = temp.parent;
+    }
+    return depth;
+};
+
+const sortItemsByZIndex = function (a, b) {
+    if (a === null || b === null) {
+        // Incomparable
```

Comment:
_/warning: irrelevant language rabbit hole follows_

I stumbled on "incomparable" in this context, generally it means "something that is so good that it is beyond comparison". However, dictionaries don't (yet) recognize "uncomparable" as a word, but there isn't really a good word that unambiguously means "cannot be compared". I guess "incommensurable" means that, but it is a mouthful and has more of a philosophical bent than is necessary here, and it is easily mistaken for "incommensurate", which is completely different.

Reply:
"Not comparable" can also work although it's two words, I counted.

Reply:
would you say "not COMPrable" or "not comPAIRable"?

I feel like saying "inCOMPrable" means beyond compare, and "incomPAIRable" means it doesn't makes sense/is impossible to compare.

Reply:
![image](https://user-images.githubusercontent.com/2855464/34166563-3a0b4040-e4ad-11e7-9170-ba57488000cd.png)

Reply:
But that example sentence is super ambiguous! What does it even mean? Is censorship worse now, or less bad?!